tmail-backend/blob/secondary-blob-store/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>tmail-backend</artifactId>
+        <groupId>com.linagora.tmail</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>secondary-blob-store</artifactId>
+    <name>Twake Mail :: Blob :: SecondaryBlobStore</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-backends-cassandra</artifactId>
```

Comment:
This dependency should not be a hard dependency

Reply:
What is a hard dependency? Is there a soft dependency?

Reply:
Well definitly should not be included in dependencies of the maven module.

> Is there a soft dependency?

scope test or provided can be seen as "softer dependencies.